tox.ini

```diff
@@ -21,6 +25,10 @@ deps =
     django110: Django>=1.10,<1.11
     django111: Django>=1.11,<2.0
     django20: Django>=2.0,<2.1
+    django21: Django>=2.1,<2.2
```

Comment:
I've added django21 here for exhaustivity but I don't think we should as it's no longer a supported version (the only active LTS on 2.x is 2.2)